### PR TITLE
udpate README description

### PR DESCRIPTION
**Pull request by mydmdm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,17 +64,28 @@ OpenPAI is a most complete solution for deep learning, support virtual cluster,
 
 OpenPAI manages computing resources and is optimized for deep learning. Through docker technology, the computing hardware are decoupled with software, so that it's easy to run distributed jobs, switch with different deep learning frameworks, or run other kinds of jobs on consistent environments.
 
-As OpenPAI is a platform, there might be two different roles who are involved. One is the cluster administrator, and the other is the cluster user. We provide end-to-end manuals for different roles.
+As OpenPAI is a platform, there are typically two different roles
```
Comment (hzy46) [inline README.md:67]: Add a ":"? Like `Two different roles:`

File: README.md [hunk 2]
```diff
@@ -64,17 +64,28 @@ OpenPAI is a most complete solution for deep learning, support virtual cluster,
...
 
-### Manual for cluster administrators
+- **Cluster users** are the consumer of the cluster's computing resources. According to the deployment scenarios, cluster users could be researchers of Machine Learning and Deep Learning, data scientists, lab teachers and students and so on.
```
Comment (hzy46) [inline README.md:69]: consumer -> consumers; 

lab teachers and students and so on -> lab teachers, students, and so on.

File: README.md [hunk 3]
```diff
@@ -64,17 +64,28 @@ OpenPAI is a most complete solution for deep learning, support virtual cluster,
 
 OpenPAI manages computing resources and is optimized for deep learning. Through docker technology, the computing hardware are decoupled with software, so that it's easy to run distributed jobs, switch with different deep learning frameworks, or run other kinds of jobs on consistent environments.
 
-As OpenPAI is a platform, there might be two different roles who are involved. One is the cluster administrator, and the other is the cluster user. We provide end-to-end manuals for different roles.
+As OpenPAI is a platform, there are typically two different roles
 
-### Manual for cluster administrators
+- **Cluster users** are the consumer of the cluster's computing resources. According to the deployment scenarios, cluster users could be researchers of Machine Learning and Deep Learning, data scientists, lab teachers and students and so on.  
+- **Cluster administrators** are the owner and maintainer of computing resources. The administrators are responsible for the deployment and availability of the cluster.
```
Comment (hzy46) [inline README.md:70]: owner and maintainer -> owners and maintainers

File: README.md [hunk 4]
```diff
@@ -64,17 +64,28 @@ OpenPAI is a most complete solution for deep learning, support virtual cluster,
...
+### For cluster administrators
 
-The manual also covers topics about how to manage user/storage/virtual cluster, how to add or remove nodes, and how to uninstall PAI. Please refer to [here](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/README.html) for the full contents.
+The [admin manual](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/README.html) is a comprehensive guide for cluster administrators, it covers (but not limited to) the following contents
```
Comment (hzy46) [inline README.md:76]: Add a ":" ?

File: README.md [hunk 5]
```diff
@@ -64,17 +64,28 @@ OpenPAI is a most complete solution for deep learning, support virtual cluster,
...
+
+    *If there is any question during deployment, please check [installation FAQs and troubleshooting](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/installation-faqs-and-troubleshooting.html) first. If it is not covered yet, refer to [here](#get-involved) to ask question or submit an issue.*
+
+- **Basic cluster management**. Through the Web-portal and a command-line tool `paictl`, administrators could complete [cluster managements](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/basic-management-operations.html), such as [adding (or removing) nodes](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/basic-management-operations.html#basic-management-operations), [monitoring nodes and services](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/basic-management-operations.html#management-on-webportal), and [storages setup and permission control](https://openpai.readthedocs.io/en/latest/manual/cluster-admin/how-to-set-up-storage.html).
```
Comment (hzy46) [inline README.md:82]: The link for `adding (or removing) nodes` should be `https://openpai.readthedocs.io/en/latest/manual/cluster-admin/how-to-add-and-remove-nodes.html` instead.

File: README.md (hunk 1 above)
Comment (mydmdm) [reply README.md]: done

**Pull request merged**